pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -5769,6 +5769,16 @@ fn test_multiple_validators_from_an_entity() {
                 Error::<Test>::HitIntendedValidatorCount
             );
 
+            // Can change commission of existing nodes
+            // even after hitting the operator node limit
+            ValidatorCommissionCap::put(Perbill::one());
+            assert_ok!(Staking::validate(
+                Origin::signed(51),
+                ValidatorPrefs {
+                    commission: Perbill::one(),
+                }
+            ));
```

Comment:
```suggestion
            let commission = Perbill::one();
            ValidatorCommissionCap::put(commission);
            assert_ok!(Staking::validate(
                Origin::signed(51),
                ValidatorPrefs { commission }
            ));
```
Just to clarify the relationship under test.